src/index.js

```diff
@@ -9,87 +9,93 @@
 import { fork } from 'child_process';
 import { config as hathorLibConfig } from '@hathor/wallet-lib';
 
-import config from './config';
 import createApp from './app';
 import { EVENTBUS_EVENT_NAME, notificationBus } from './services/notification.service';
 import version from './version';
-
-if (config.enabled_plugins && config.enabled_plugins.length > 0) {
-  // There are configured plugins, we should start the child process
-
-  // We will pass the argv we receive since the config module may need these arguments
-  // options.silent will pipe child stdout to main process
-  // options.env will spawn child with the same environment as main process (for config)
-  const child = fork(
-    `${__dirname}/plugins/child.js`,
-    process.argv.slice(2),
-    { silent: true, env: process.env },
-  );
-  console.log(`child process started with pid ${child.pid}`);
-
-  process.on('exit', () => {
-    if (child.connected || !child.killed) {
-      console.log('disconnecting from child.');
-      child.disconnect();
-    }
-  });
-
-  // This is to unify logs from child and main process.
-  child.stdout.on('data', console.log);
-
-  // Pipe child stderr to stdout.
-  child.stderr.on('data', console.error);
-
-  child.on('error', err => {
-    console.error(`child process error: ${err.message}`);
-  });
-
-  child.on('disconnect', (code, signal) => {
-    console.log(`child process disconnected from IPC channel with (${code} and ${signal})`);
-    // Killing child just in case it has not yet died.
-    child.kill(); // SIGTERM
+import settings from './settings';
+
+async function startHeadless() {
```

Comment:
Needed to encapsulate the logic inside an async method since we need to `await` the settings setup below.
The setup is required to be async as well since it makes a dynamic import which can only be async.